ubuntu_image/helpers.py

```diff
@@ -214,9 +214,16 @@ def mkfs_ext4(img_file, contents_dir, label='writable',
     Ubuntu 16.04, which has e2fsprogs 1.42.X without the -d flag.  In
     that case, we have to sudo loop mount the ext4 file system and
     populate it that way.  Which sucks because sudo.
+    NOTE Unfotunately we cannot use fakeroot for classic, as that changes the
```

Comment:
Ok, now this is a bug we were not aware of before. I have just checked in a local test and this is right. Therefore I would actually prefer for this not to be part of this PR but a separate one instead. I think I like this fix you proposed - we generally switched to using fakeroot-sysv here because we didn't want snap builds to rely on sudo, so in this case we should be good.

But as said, I would really prefer for this to be a separate PR. It's not required for the work here and it's a bug already existing without the --filesystem parts added. Could you do that?

Reply:
Done: https://github.com/CanonicalLtd/ubuntu-image/pull/158